### PR TITLE
Feature 'Access To The AddIoEadmin page' Tion Cada

### PR DESCRIPTION
**Pull request by TionCada**
dev
## CLICKUP

* [CLICKUP task](https://app.clickup.com/t/5n4kmv)

## Code reviewers
@ostannja 
@TaraskiN85 
@OleksandrKyrylyuk 

## Summary of issue

Super Admin can now access "Add IoE admin" page after deleting current IoE admin, test coverage => 95%.

## Summary of change

Transfered all current functionality from "University admins" page to the "Universities" page

## Testing approach

testing-library

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by TaraskiN85 — APPROVED** [review]
Don't forget about spinner

**Comment by TionCada** [comment]
> 
> 
> Don't forget about spinner
Already done 👌

**Review by OleksandrKyrylyuk — APPROVED** [review]

File: src/pages/superAdminPages/universityListPage/universityListPage.module.scss [hunk 1]
```diff
@@ -1,5 +1,12 @@
 @import '../../../styles/index.scss';
 
+.noContentContainer {
+  display: flex;
```
Comment (ostannja) [inline src/pages/superAdminPages/universityListPage/universityListPage.module.scss]: try to use mixins for positioning (src/styles/mixins.scss)

**Review by ostannja — APPROVED** [review]

File: src/pages/superAdminPages/universityListPage/universityListPage.module.scss (hunk 1 above)
Comment (TionCada) [reply src/pages/superAdminPages/universityListPage/universityListPage.module.scss]: done ✔️

**Pull request merged**